dev/reference/tools/tidb-lightning/web.md

```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB Lightning Web Interface
+summary: Control TiDB Lightning through the web interface.
+category: reference
+---
+
+# TiDB Lightning Web Interface
+
+TiDB Lightning provides a webpage for viewing import progress and performs some simple task management.
+This is called *Server mode*.
+
+To enable Server mode, either start `tidb-lightning` with the `--server-mode` flag
+
+```sh
+./tidb-lightning --server-mode --status-addr :8289
+```
+
+or set the `lightning.server-mode` setting in the configuration file.
+
+```toml
+[lightning]
+server-mode = true
+status-addr = ':8289'
+```
+
+After TiDB Lightning is launched, visit `http://127.0.0.1:8289` to control the program (the actual URL depends on the `status-addr` setting).
+
+In Server mode, TiDB Lightning does not start running immediately.
+Rather, users submit (multiple) *tasks* via the web interface to import data.
+
+## Front page
+
+![Front page of the web interface](/media/lightning-web-frontpage.png)
+
+Functions of the title bar, from left to right:
+
+| Icon | Function |
+|----|----|
```

Comment:
We use left-alignment in documentation. Please update all tables in this PR to keep them consistent. Thanks!

```suggestion
|:----|:----|
```